rust/protocol/src/identity_key.rs

```diff
@@ -11,12 +13,14 @@ use std::convert::TryFrom;
 
 use prost::Message;
 
+/// Wrapper for [PublicKey].
 #[derive(Debug, PartialOrd, Ord, PartialEq, Eq, Clone, Copy)]
 pub struct IdentityKey {
     public_key: PublicKey,
 }
 
 impl IdentityKey {
+    /// Create a new instance.
```

Comment:
This doesn't add any information that isn't obvious from the function signature. I suggest to either remove this and similar comments or add more information.

Reply:
Thanks! I have removed all such comments, especially those in `inmem.rs`.